src/assets/js/scope.js

```diff
@@ -11,15 +11,21 @@ class Scope {
 
   update(user) {
     const {
+      uuid: { value: uuid = '00000000-0000-0000-0000-000000000000' } = {},
+      primaryUsername: { value: username = '' } = {},
       staffInformation: { staff: { value: isStaff = false } = {} } = {},
       accessInformation: {
         mozilliansorg: { values: mozilliansorgGroups = {} } = {},
         ldap: { values: ldapGroups = {} } = {},
       } = {},
     } = user || {};
+    this.username = username;
+    this.uuid = uuid;
+    this.isLoggedIn = Boolean(username);
     this.isStaff = isStaff;
     this.isNdaed = 'nda' in (mozilliansorgGroups || {});
     this.isLdap = Boolean(ldapGroups);
+    this.isGroupCreator = 'group_creators' in (mozilliansorgGroups || {});
```

Comment:
This is fine for now, but I'd like to see this abstracted a bit more away. Like maybe have a config file of keys or something. A static string like this (and like the nda one)  seems like it could use some refactoring. Not necessary for now, but worth mentioning.

Reply:
In general I agree. In this case I'd argue that this should be encapsulated in a single place. Making this a constant in this file which in return is only referenced once, again in this file, doesn't feel right.
Anyhow, super weak opinion: if you prefer me moving that out I'm fine. Not sure though what a config file should look like. That would be a simple .js file with constants I guess since all of this is static.

Reply:
No it's fine. I also prefer a source of truth, but this is just piggy backing off an issue that already existed. I'm fine getting this out so long as we create stories around fixing this.

Reply:
https://jira.mozilla.com/browse/IAM-541